### DISCUSSION TITLE
PLEASE HELP

### DISCUSSION THREAD

**Discussion by NahuelDK** [chromiecraft/chromiecraft#2878] [General]
Hello, could you please give me a solution? I can't get into Chromiecraft it tells me the following: This WoW account has been closed and is no longer available.
A discord admin told me it was bot usage. but I didn't know what it was until today, I never insulted anyone or stole items, could you please give me a solution? I don't want to leave Chromiecraft.

**Comment by 55Honey** [comment]
Hello NahuelDk,

As you we're ready told by another Staff member, your accounts were found to be controlled by an automated 3rd party software.

We do not take such infractions lightly and we do not accept appeals for doubtless proven cases like yours.

Due to the harm you inflicted to our project, you will never be welcome to play on ChromieCraft anymore.

**Reply by NahuelDK** [reply]
Hello 55 Miel, I only had the atlasloot addon in my world of warcraft installed, I'm a chromiecraft user since September and I never had any problems, I think it's a mistake, I don't know, I only have one account I don't use multiaccounts or anything Last time I connected I was doing dungeons with a friend, at no time did I use bots or any kind of hack. please I don't want to leave chromiecraft I have many friends and I have a great time.

**Comment by 55Honey** [comment]
Closing it due to ~~https://github.com/chromiecraft/chromiecraft/discussions/2879~~ https://github.com/chromiecraft/chromiecraft/discussions/2888 being opened.

**Comment by 55Honey** [comment]
Hello @NahuelDK,

We've looked into the chain of events once more and apparently, there was a mistake on my end. We have mixed up numbers, which resulted in your account being punished for a thing they never did.
From the data that I have checked just now, you did not use any 3rd party software and you did not violate our rules.

I want to say sorry for this ban. It was undeserved and only my own mistake.
Your account is going to be unbanned on short notice.

**Comment by 55Honey** [comment]
Solved in https://github.com/chromiecraft/chromiecraft/discussions/2878
